packages/xsnap/test/test-xs-js.js

```diff
@@ -168,6 +168,7 @@ test('round-trip byte sequences via JSON including string literals', async t =>
       .fill(0)
       .map((_, i) => `\\x${i.toString(16).padStart(2, '0')}`)
       .join('');
+    // @ts-expect-error special case
```

Comment:
special case of what?

Reply:
"special case" of evaluation I suppose. any better comment?

Reply:
I guess I'm not sure what the tsc error is in the first place

Reply:
pasted the error message

Reply:
ah yeah that makes a lot of sense now. tsc doesn't know the difference between direct and indirect eval.